test/ex_machina/ecto_test.exs

```diff
@@ -90,9 +131,10 @@ defmodule ExMachina.EctoTest do
 
     user = ExMachina.Ecto.assoc(MyApp.EctoFactories, attrs, :user)
 
-    created_user = %{id: 3, name: "John Doe", admin: false}
+    vals = %{name: "John Doe", admin: false}
+    created_user = struct(MyApp.User, vals) |> Map.put(:id, 1)
     assert user == created_user
-    assert_received {:created, ^created_user}
+    assert_received {:created, %MyApp.User{}}
```

Comment:
Yeah that is totally understandable! Good points. I agree, if we need to test that scenario we can always change it later. Good work on this!